src/url.js

```diff
@@ -54,7 +54,8 @@ export const SOURCE_ORIGIN_PARAM = '__amp_source_origin';
  * @return {string} origin
  */
 export function getWinOrigin(win) {
-    return win.origin || parseUrl(win.location.href).origin;
+  return (
+    win.origin || win.origin==-"null" || parseUrl(win.location.href).origin);
```

Comment:
I'm not quite understanding what this means. Please clarify. Why `==-` (minus in the end)?

Reply:
lol - just fixed this .

Reply:
Done